sql/migrations/20201010150010_world.sql

```diff
@@ -0,0 +1,123 @@
+(496803, 0, 15, 20680, 0, 0, 0, 0, 0, 4, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 'Lady Jaina Proudmoore - Cast Spell');
+INSERT INTO `creature_ai_events` (`id`, `creature_id`, `condition_id`, `event_type`, `event_inverse_phase_mask`, `event_chance`, `event_flags`, `event_param1`, `event_param2`, `event_param3`, `event_param4`, `action1_script`, `action2_script`, `action3_script`, `comment`) VALUES (496803, 4968, 0, 0, 0, 100, 1, 8000, 12000, 25000, 28000, 496803, 0, 0, 'Lady Jaina Proudmoore - Cast Blizzard');
+
+INSERT INTO `creature_ai_scripts` (`id`, `delay`, `command`, `datalong`, `datalong2`, `datalong3`, `datalong4`, `target_param1`, `target_param2`, `target_type`, `data_flags`, `dataint`, `dataint2`, `dataint3`, `dataint4`, `x`, `y`, `z`, `o`, `condition_id`, `comments`) VALUES
+(496804, 0, 15, 20679, 0, 0, 0, 0, 0, 4, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 'Lady Jaina Proudmoore - Cast Spell');
+INSERT INTO `creature_ai_events` (`id`, `creature_id`, `condition_id`, `event_type`, `event_inverse_phase_mask`, `event_chance`, `event_flags`, `event_param1`, `event_param2`, `event_param3`, `event_param4`, `action1_script`, `action2_script`, `action3_script`, `comment`) VALUES (496804, 4968, 0, 0, 0, 100, 1, 5000, 9000, 15000, 17000, 496804, 0, 0, 'Lady Jaina Proudmoore - Cast Fire Blast');
+
+INSERT INTO `creature_ai_scripts` (`id`, `delay`, `command`, `datalong`, `datalong2`, `datalong3`, `datalong4`, `target_param1`, `target_param2`, `target_type`, `data_flags`, `dataint`, `dataint2`, `dataint3`, `dataint4`, `x`, `y`, `z`, `o`, `condition_id`, `comments`) VALUES
+(496805, 0, 15, 20681, 0, 0, 0, 0, 0, 0, 4, 0, 0, 0, 0, 0, 0, 0, 0, 0, 'Lady Jaina Proudmoore - Cast Spell');
+INSERT INTO `creature_ai_events` (`id`, `creature_id`, `condition_id`, `event_type`, `event_inverse_phase_mask`, `event_chance`, `event_flags`, `event_param1`, `event_param2`, `event_param3`, `event_param4`, `action1_script`, `action2_script`, `action3_script`, `comment`) VALUES (496805, 4968, 0, 0, 0, 100, 1, 4000, 5000, 45000, 55000, 496805, 0, 0, 'Lady Jaina Proudmoore - Cast Summon Water Elementals');
+
+INSERT INTO `creature_ai_scripts` (`id`, `delay`, `command`, `datalong`, `datalong2`, `datalong3`, `datalong4`, `target_param1`, `target_param2`, `target_type`, `data_flags`, `dataint`, `dataint2`, `dataint3`, `dataint4`, `x`, `y`, `z`, `o`, `condition_id`, `comments`) VALUES
+(496806, 0, 15, 20692, 8, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 'Lady Jaina Proudmoore - Cast Spell');
+INSERT INTO `creature_ai_events` (`id`, `creature_id`, `condition_id`, `event_type`, `event_inverse_phase_mask`, `event_chance`, `event_flags`, `event_param1`, `event_param2`, `event_param3`, `event_param4`, `action1_script`, `action2_script`, `action3_script`, `comment`) VALUES (496806, 4968, 0, 9, 0, 100, 1, 0, 50, 2000, 3500, 496806, 0, 0, 'Lady Jaina Proudmoore - Cast Fireball');
+UPDATE `creature_template` SET `ai_name` = 'EventAI' WHERE `entry` = 4968;
+
+
+-- Vol'jin 10540

```

Comment:
He already uses all these abilities from his spell list.

Reply:
Thanks for reviewing